### ISSUE TITLE
Wrong glyph for U+2E49 DOUBLE STACKED COMMA

### ISSUE THREAD

**Issue by dscorbett** [googlefonts/noto-fonts#1817]
### Font
NotoMono-Regular.otf
NotoSans-Regular.ttf
NotoSerif-Regular.ttf

### Where the font came from, and when
Site: https://github.com/googlefonts/noto-fonts/blob/bf20559450ec75aec7a646b208343540a4496262/phaseIII_only/unhinted/otf/NotoMono/NotoMono-Regular.otf
Site: https://github.com/googlefonts/noto-fonts/blob/bf20559450ec75aec7a646b208343540a4496262/phaseIII_only/hinted/ttf/NotoSans/NotoSans-Regular.ttf
Site: https://github.com/googlefonts/noto-fonts/blob/bf20559450ec75aec7a646b208343540a4496262/phaseIII_only/hinted/ttf/NotoSerif/NotoSerif-Regular.ttf
Date: 2020-08-01

### Font version
Noto Mono: Version 2.004
Noto Sans: Version 2.003; ttfautohint (v1.8.2)
Noto Serif: Version 2.003; ttfautohint (v1.8.2)

### Issue
U+2E49 DOUBLE STACKED COMMA should look like two stacked commas.

In Noto Serif, it looks like two stacked old-fashioned Cyrillic commas. Since this is not an old-fashioned Cyrillic font, that is inappropriate. The representative glyph in the Unicode code chart uses that style because U+2E49’s proposal ([L2/15-173R2](https://www.unicode.org/L2/L2015/15173r2-add-typicon.pdf)) was focused on typicon symbols. However, the book [<i>Ключ к церковному уставу</i>/<i>Клю́чь къ церко́вномꙋ оу҆ста́вꙋ</i>](http://seminaria.ru/syrnikov-n-s-klyuch-k-tserkovnomu-ustavu-chastnoye-izdaniye-po-zakazu-n-s-syrnikova-dlya-upotrebleniya-sego-sochineniya-tolko-v-yedinovercheskikh-tserkvakh-m-1910)  cited in that proposal [describes this symbol as “двѣ̀ запѧты́хъ че́рныⷯ”](https://227704.selcdn.ru/mrps-library/texts/science/liturgica/ustav_key/znak_slujby.pdf#page=2), i.e. two black commas, implying that it was conceived of simply as two commas, and that its glyph should therefore be harmonized with the glyph of the comma.

In Noto Mono and Noto Sans, it looks like two stacked semicircles. Where did such a design come from? It should look like two commas in these fonts too.

### Character data
⹉,
U+2E49 DOUBLE STACKED COMMA
U+002C COMMA

### Screenshot
![⹉,](https://user-images.githubusercontent.com/1124347/90317442-9e290c00-def7-11ea-8806-a7810d06d45a.png)![⹉,](https://user-images.githubusercontent.com/1124347/90317443-9e290c00-def7-11ea-988d-c45a8d47f119.png)![⹉,](https://user-images.githubusercontent.com/1124347/90317444-9e290c00-def7-11ea-95dc-148f1fbd07f2.png)

**Comment by moyogo** [comment]
@typiconman do you have any comment on this?

**Comment by typiconman** [comment]
Yes, it should look like two stacked commas, and not two semicircles.

**Comment by nizarsq** [comment]
<img width="533" alt="Screen Shot 2020-08-18 at 2 57 08 PM" src="https://user-images.githubusercontent.com/22158882/90569646-1810fd80-e163-11ea-923b-2f248afc4ec6.png">
